docs/BR.md

```diff
@@ -1739,142 +1737,907 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+
+| __Access Method__ | __OID__            | __Presence__ | __Access Location__         | __Description__ |
+| --                | --                 | -            | ----                        | ---             |
+| `id-ad-ocsp`      | 1.3.6.1.5.5.7.48.1 | MUST         | `uniformResourceIdentifier` | A HTTP URL of the Issuing CA's OCSP responder. |
+| `id-ad-caIssuers` | 1.3.6.1.5.5.7.48.2 | SHOULD       | `uniformResourceIdentifier` | A HTTP URL of the Issuing CA's certificate. |
+| Any other value   | -                  | MUST NOT     | -                           | -               |
+
+##### 7.1.2.6.8 Basic Constraints
+
+| __Field__           | __Description__ |
+| ---                 | ------- |
+| `cA`                | MUST be FALSE |
+| `pathLenConstraint` | MUST NOT be present |
+
+##### 7.1.2.6.9 Certificate Policies
+
+| __Field__                  | __Presence__ | __Description__ |
+| ---                        | -            | ------          |
+| `certificatePolicies`      |              |                 |
+| \ \ **1**                  | MUST         | The first `PolicyInformation` present in the `certificatePolicies`. |
+| \ \ \ \ `policyIdentifier` |              | The Reserved Certificate Policy Identifier (see [Section 7.1.6.1](#7161-reserved-certificate-policy-identifiers)) associated with the given Subscriber Certificate type (see [Section 7.1.2.6.1](#71261-subscriber-certificate-types). |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **2+**                 | MAY          | The Issuing CA MAY include additional `PolicyInformation` values that meet the following profile: |
+| \ \ \ \ `policyIdentifier` | MUST         | An identifier documented by the Issuing CA in its Certificate Policy and/or Certification Practice Statement. |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **3**                  | MUST NOT     | The Issuing CA MUST NOT include any additional `PolicyInformation` values that do not meet the above profile. |
+
+If the `policyQualifiers` is permitted and present within a `PolicyInformation` field, it MUST be formatted as follows:
+
+| __Qualifier ID__                     | __Presence__ | __Field Type__ |  __Contents__ |
+| ---                                  | -            | -              | -----         |
+| `id-qt-cps` (OID: 1.3.6.1.5.5.7.2.1) | MAY          | `IA5String`    | The HTTP or HTTPS URL for the Issuing CA's Certificate Policies, Certification Practice Statement, Relying Party Agreement, or other pointer to online policy information provided by the Issuing CA. |
+| Any other qualifier                  | MUST NOT     | -              | -             |
+
+##### 7.1.2.6.10 Extended Key Usage
+
+| __Key Purpose__         | __OID__           | __Presence__ |
+| ----                    | ----              | -            |
+| `id-kp-serverAuth`      | 1.3.6.1.5.5.7.3.1 | MUST         |
```

Comment:
An explicit requirement to say serverAuth is required is new and I totally understand the discussions that if you want client auth only you should/must not use the webPKI for that.  Given this might be perceived as a change and impact some customers, should we assign a specific effective date for this (and other changes) so we can ballot the profiles work and make the spec immediately effective, but have specific dates for the items that need them?

Reply:
Could you describe a little more what you believe would be impacted? Perhaps an example crt.sh link?

I don't think your proposed fix is correct: a certificate with `id-kp-clientAuth` only is not one that should be issued for server auth, and the transition is only going to muddy those waters. I'm open to explicitly having a certificate profile for this, but I think it'd be useful if you could provide an example cert or certs to understand what this looks like today.

Reply:
Just following up on this @dougbeattie - You seemed OK on our most recent call, but did want to touch base to make sure before closing this out as resolved.